dkg/dkg.go

```diff
@@ -73,6 +75,14 @@ func Run(ctx context.Context, conf Config) (err error) {
 		return err
 	}
 
+	// Check if keymanager address is reachable.
+	if conf.KeymanagerAddr != "" {
+		cl := keymanager.New(conf.KeymanagerAddr)
+		if err = cl.VerifyConnection(ctx); err != nil {
+			return err
```

Comment:
what does this error look like to users? please paste a screenshot

Reply:
<img width="1891" alt="Screenshot 2023-01-24 at 15 29 16" src="https://user-images.githubusercontent.com/43061995/214266708-d0b38fad-e8e1-494a-bfbe-7e835f6d2c5a.png">

Should we have `cannot ping keymanager address` in error log?

Reply:
suggest: `return errors.Wrap(err, "verify keymanager address"`